package/CHANGELOG

```diff
@@ -39,6 +40,7 @@ Fixes
  * Fix groups.py doctests using sphinx directives (Issue #3925, PR #4374)
 
 Enhancements
+ * Handling of Pathlib.Path in SingleFrameReaderBase, Topology and Universe (Issue #3937)
```

Comment:
This is a bit vague and doesn't actually tell me what is fixed or what is now possible.  e.g. Is it that `pathlib.Path`s are now correctly handled?

Reply:
Correct, originally it was only targeting pathlib object handling of SingleFrameReaderBase in specific cases, but with the adjustment of @hmacdope it would cover `Pathlib.Path`s for all cases if the errros that appear currently due to the changes in the `base.py` in topology and also the `base.py` in coordinates, which could be fixed with 

`elif hasattr(filename, 'topology'):`
 `    self.filename = filename`
 
 but this would just exclude the cases of `topology` so I am not sure if this would fix it and cover then all the  `Pathlib.Path` cases